src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -534,7 +535,7 @@ public void testUserApiForNonSuperAdmin() throws Exception {
 
         // Put read only users
         response = rh.executePutRequest("/_opendistro/_security/api/internalusers/sarek", "[{ \"op\": \"add\", \"path\": \"/sarek/description\", \"value\": \"foo\" }]", new Header[0]);
-        Assert.assertEquals(HttpStatus.SC_FORBIDDEN, response.getStatusCode());
+        Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
```

Comment:
Why is this bad request ? it is updating read only user and password field is not present

Reply:
Moving Line #56 - 57 down [here](https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/CredentialsValidator.java#L56) caused NotXContentException on Line #6. The method validate() still returns false for this use case, but bad request instead of forbidden.

Reply:
Fixed the test.